xml/gl.xml

```diff
@@ -38078,6 +38078,18 @@ typedef unsigned int GLhandleARB;
             <command name="glReadnPixels"/>
         </require>
         <require profile="compatibility" comment="Reuse GL_ARB_robustness">
+            <enum name="GL_COLOR_TABLE"/>
```

Comment:
I don't think this is the correct place for these enum usages, given that they weren't declared in GL_ARB_robustness.

Reply:
If they arent declared here they are not in this feature at all, that is the entire problem. Unless you are talking about just moving them to another <require> tag

Reply:
Yeah I'm just on about they don't really belong in this `require` block. Feel free to setup another require block.